icicle/utils/mont.cu

```diff
@@ -0,0 +1,35 @@
+#include "../curves/curve_config.cuh"
+#include "device_context.cuh"
+#include "mont.cuh"
+
+namespace mont {
+  extern "C" int ScalarConvertMontgomery(
+    curve_config::scalar_t* d_inout, size_t n, unsigned char is_into, device_context::DeviceContext& ctx)
```

Comment:
Can we just use bool for is_into here? Bool is FFI safe: https://stackoverflow.com/a/47705543